src/applications/vaos/appointment-list/components/AppointmentsPage/index.jsx

```diff
@@ -192,11 +192,11 @@ function mapStateToProps(state) {
     pendingStatus: state.appointments.pendingStatus,
     futureStatus: selectFutureStatus(state),
     cancelInfo: getCancelInfo(state),
-    showPastAppointments: vaosPastAppts(state),
-    showScheduleButton: vaosRequests(state),
-    showCommunityCare: vaosCommunityCare(state),
-    showDirectScheduling: vaosDirectScheduling(state),
-    showExpressCare: vaosExpressCare(state),
+    showPastAppointments: selectFeaturePastAppointments(state),
```

Comment:
I think updating these props to follow a convention like `featurePastAppoinments` instead of `showPastAppointments` would also be good, but I'll leave that to a separate PR, because it's messier.